cylc/flow/cfgspec/workflow.py

```diff
@@ -1090,16 +1090,17 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                 )
             )
             Conf('execution retry delays', VDR.V_INTERVAL_LIST, None, desc=f'''
```

Comment:
Should do something similar for `submission retry delays`?

Reply:
Done